### PR TITLE
refine the doc for split Op test=develop

### PR DESCRIPTION
**Pull request by wangchaochaohu**
split Op
image
image

### CODE REVIEW

**Review by lanxianghit — APPROVED** [review]
LGTM

File: doc/fluid/api_cn/layers_cn/split_cn.rst [hunk 1]
```diff
@@ -3,54 +3,58 @@
...
     - **name** (str，可选) - 具体用法请参见 :ref:`api_guide_Name` ，一般无需设置，默认值为None。
 
 返回：分割后的Tensor列表。
 
-返回类型：列表(Variable(Tensor|LoDTensor))，数据类型为int32，int64，float32，float64。
 
 抛出异常：
-    - :code:`TypeError`：``num_or_sections`` 不是int、list 或 tuple。
-    - :code:`TypeError`：``dim`` 不是 int 或 Variable。
+    - :code:`TypeError`：``x`` 的数据类型不是bool、float16、float32、float64、int32或int64时 。
```
Comment (TCChenlong) [inline doc/fluid/api_cn/layers_cn/split_cn.rst]: input吧？

File: doc/fluid/api_cn/tensor_cn/split_cn.rst [hunk 2]
```diff
@@ -2,39 +2,58 @@
 split
 -------------------------------
 
-.. py:function:: paddle.tensor.split(input, num_or_sections, dim=-1, name=None)
+.. py:function:: paddle.tensor.split(x, num_or_sections, axis=0, name=None)
 
 :alias_main: paddle.split
-:alias: paddle.split,paddle.tensor.split,paddle.tensor.manipulation.split
-:update_api: paddle.fluid.layers.split
+:alias: paddle.tensor.split, paddle.tensor.manipulation.split
```
Comment (TCChenlong) [inline doc/fluid/api_cn/tensor_cn/split_cn.rst]: 2.0也不用写alias 这两行alias也麻烦删了吧

**Review by TCChenlong — APPROVED** [review]
LGTM

File: doc/fluid/api_cn/layers_cn/split_cn.rst (hunk 1 above)
Comment (wangchaochaohu) [reply doc/fluid/api_cn/layers_cn/split_cn.rst]: Done

File: doc/fluid/api_cn/tensor_cn/split_cn.rst (hunk 2 above)
Comment (wangchaochaohu) [reply doc/fluid/api_cn/tensor_cn/split_cn.rst]: Done

**Pull request merged**